openmls/src/treesync/node/leaf_node.rs

```diff
@@ -382,6 +384,26 @@ impl TlsDeserializeTrait for LeafNodeTbs {
     }
 }
 
+// TODO: Name up to debate.
+#[derive(Debug, Clone, PartialEq, Eq, TlsSize, TlsDeserialize)]
+pub struct StateUnvalidated;
```

Comment:
This is the initial leaf node state. No validations were done to this leaf node.